library/rtsp/src/main/java/com/google/android/exoplayer2/source/rtsp/RtspMediaPeriod.java

```diff
@@ -96,19 +97,21 @@
 
   /**
    * Creates an RTSP media period.
-   *
    * @param allocator An {@link Allocator} from which to obtain media buffer allocations.
    * @param rtpDataChannelFactory A {@link RtpDataChannel.Factory} for {@link RtpDataChannel}.
    * @param uri The RTSP playback {@link Uri}.
    * @param listener A {@link Listener} to receive session information updates.
    * @param userAgent The user agent.
+   * @param debugLoggingEnabled Whether to log RTSP messages.
+   * @param socketFactory A socket factory for {@link RtspClient}'s connection.
```

Comment:
super-nit: could you swap the order of these two lines to match the parameter order?